explainers/Sustainability/explainer.md

```diff
@@ -1,49 +1,56 @@
+The carbon impact of the internet, including the hardware, software, and systems that go into creating it, produces about [3.7%]( https://www.bbc.com/future/article/20200305-why-your-internet-habits-are-not-as-clean-as-you-think) of global carbon emissions. This is similar to the percentage of global emissions from the [airline industry](https://www.bbc.com/future/article/20200305-why-your-internet-habits-are-not-as-clean-as-you-think). Developers and the public are becoming increasingly more aware and concerned about this issue. Developers are increasingly interested in understanding **what factors impact a web app's sustainability and how to improve it's sustainability.** Consequently, tools such as [Website Carbon](https://www.websitecarbon.com/), [Ecograder](https://ecograder.com/), [EcoPing](https://ecoping.earth/blog/core-web-vitals-and-sustainability), and [Beacon](https://digitalbeacon.co/) have been created to calculate the amount of CO2 a web app emits and provide recommendations to improve the sustainability of a web app.
 ## Goals
-Adding sustainability features to Edge DevTools will raise awareness about the alarming carbon impact of web apps and the internet and help developers measure and improve the sustainability of their web apps by
+Adding sustainability features to DevTools will raise awareness about the alarming carbon impact of web apps and the internet and help developers measure and improve the sustainability of their web apps by
 -	Providing proxy web app sustainability measurements
 -	Highlighting common factors that reduce the sustainability of web apps by showing or explaining how these factors decrease performance and increase energy consumption
 -	Indicating to developers what actions they’re currently taking, such as performance improvements, that also improve web app sustainability
 -	Educating developers on new actions they can take that are purely beneficial for sustainability.
 ## Solution
-Summary of features we’re investigating adding to DevTools:
+### Summary 
 -	Label existing issues in the Issues panel that are beneficial to sustainability and energy efficiency 
--	Create a “Sustainability” tab that provides metrics on the overall energy consumption of the web app, the impact third parties have on energy and performance, and a list of best practices to improve sustainability
+- Create a “Sustainability” tab that is a hub for sustainability-related metrics and provides insights about web app energy consumption, the type of energy used to power a web app, third parties' impact on performance and energy, and best practices to improve sustainability. 
```

Comment:
This is another minor markdown formatting tip: please try to use either only spaces or only tabs throughout the code. The first bullet point here seems to use a tab, whereas the second uses a space. It may not look different for you in your text editor (depending on how tabs are set up), but it looks different on github when doing a review, and it may look different in other people's editors, and hence make it a little bit harder to read.